samples/unit-tests/3d/series-pie/demo.js

```diff
@@ -348,5 +351,11 @@ QUnit.test(
             height < point.graphic.out.getBBox(true).height,
             'Updating series.depth should change slice\'s depth (#12515).'
         );
-    }
-);
\ No newline at end of file
+
+        assert.ok(
+            chart.series[0].group.oldtranslateX === chart.plotLeft &&
+            chart.series[0].group.oldtranslateY === chart.plotTop,
+            'Updating series shouldn\'t change pie position (#11928).'
```

Comment:
Do not use `assert.ok` to test values that should be equal. Instead, break this up into two `strictEqual` assertions.